wrappers/ios/demo/demo.xcodeproj/project.pbxproj

```diff
@@ -316,7 +316,7 @@
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = 25MXZ8DDDM;
```

Comment:
I guess you accidentally reverted your earlier change here..

Reply:
Yes you are right 🙈I'll take care of this tomorrow.